### PR TITLE
NT_STATUS_NOT_FOUND => name cannot be resolved.

### PR DESCRIPTION
**Pull request by SkyLined**
I used a name that could not be resolved and got this error, so I thought I would provide a better error message.

### CODE REVIEW

File: samba-backup/rootfs/scripts/precheck.sh [hunk 1]
```diff
@@ -14,6 +14,10 @@ function smb-precheck {
     if ! result=$(eval "${SMB} -c 'exit'"); then
         bashio::log.warning "$result"
 
+        # host not found
+        if [[ "$result" =~ "NT_STATUS_NOT_FOUND" ]]; then
+            bashio::log.fatal "The provided host cannot be found. Please check your config and network."
```
Comment (thomasmauerer) [inline samba-backup/rootfs/scripts/precheck.sh:19]: Please change the second sentence to "If you've specified a DNS name, please try using an IP address instead."
I guess we can make a more concrete suggestion in this case.

File: samba-backup/rootfs/scripts/precheck.sh [hunk 2]
```diff
@@ -14,6 +14,10 @@ function smb-precheck {
...
+
         # host unreachable
         if [[ "$result" =~ "NT_STATUS_HOST_UNREACHABLE" ]]; then
```
Comment (thomasmauerer) [inline samba-backup/rootfs/scripts/precheck.sh:22]: Change this to elif, otherwise the last else block will additionally execute.

**Comment by thomasmauerer** [comment]
Thanks for the PR. Please check my two little comments.

**Review by thomasmauerer — CHANGES_REQUESTED** [review]

**Comment by SkyLined** [comment]
These changes makes sense. I assume I do not need to start all over and can just make changes to my existing PR but I cannot seem to find out how to do that.

**Comment by thomasmauerer** [comment]
Yes, you can. You just need to push more commits to your patch-1 branch.

**Comment by SkyLined** [comment]
TIL. Thanks, in hindsight it always seems obvious.

**Review by thomasmauerer — APPROVED** [review]

**Comment by thomasmauerer** [comment]
Looks good, thanks.

**Pull request merged**